appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/WebServicesApplication.java

```diff
@@ -155,15 +154,8 @@ private void collectEjbEndpoints(BundleDescriptor bundleDesc) {
                 }
             }
         }
-        //For ejb webservices in war we need to get the extension descriptors
```

Comment:
Re Shingwai's comment on extension descriptor: 
This part of the code seems for handling EJB in war case. In that case, the main bundle descriptor is WebBundleDescriptor and the EjbBundleDescriptor is an extension descriptor of the main bundle descriptor. Can you run through some test cases for EJB in war scenario where the EJB is a webservice endpoint to make sure no regressions there?